src/js/media/html5.js

```diff
@@ -438,8 +415,6 @@ vjs.Html5.prototype.addRemoteTextTrack = function(options) {
     if (track.readyState >= 2) {
```

Comment:
I was hesitant to delete this bit because I didn't totally understand its intent. I'd like @gkatsev to review tomorrow when he's back.